python/taichi/testing.py

```diff
@@ -2,6 +2,17 @@
 
 
 # Helper functions
+def default_epsilon_of_arch(arch):
+    if arch == ti.opengl:
```

Comment:
```suggestion
def get_rel_eps():
    if ti.cfg.arch == ti.opengl:
```
We can actually expose this function to end-users? E.g., in `examples/renderer_util.py`.

Reply:
Done